src/core/include/openvino/op/concat.hpp

```diff
@@ -37,9 +37,11 @@ class OPENVINO_API Concat : public Op {
     std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& new_args) const override;
 
     /// \return The concatenation axis.
+    OPENVINO_DEPRECATED("This method will be removed in 2025.0")
     int64_t get_concatenation_axis() const {
         return m_concat_axis;
     }
+    OPENVINO_DEPRECATED("This method will be removed in 2025.0")
```

Comment:
lets encourage people to use get_axis/set_axis methods

Reply:
Done